reggie/ingestion/preprocessor/oklahoma_preprocessor.py

```diff
@@ -14,6 +14,21 @@
 from datetime import datetime
 import gc
 import json
+import re
+
+"""
+Todo:
+Add Important Columns and check order (currently precicnts are added before County)
+
+
+VF Columns:
+Todo: Fill out
+
+VH Columns: 
+  - VoterID
+  - ElectionDate
+  - VotingMethod 
+"""
```

Comment:
clean this up - though you can leave documentation about the ways the file is weird, if you want